### DISCUSSION TITLE
Unable to click single-select [Mar 27]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5405] [Discord Threads]
> 🙋‍♂️ **Hirslay**  
## Hello, something weird is happening. From pop-up window I cannot choose any MultiSelect option. Button is not responding. Any idea what's going on?  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1077863293336702976/image.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **Hirslay**  
#### as you can see I have plenty of interactions to choose  
![image0](https://cdn.discordapp.com/attachments/1077863293395415071/1077863570408218645/image.png)  
#  
> 💁 **o1lab**  
#### are you on firefox. there appears to eb a bug   
#### it works if you click the cell to right most  
#  
> 🙋‍♂️ **Hirslay**  
#### I was trying on opera, chrome and firefox  
#  
> 💁 **o1lab**  
#### let me chk  
#  
> 💁 **o1lab**  
#### which version are u using ?  
#  
> 💁 **o1lab**  
#### this is fine in latest version  
![image0](https://cdn.discordapp.com/attachments/1077863293395415071/1077926260086419486/Screenshot_2023-02-22_at_12.13.30.png)  
#  
> 🙋‍♂️ **Hirslay**  
#### I'm using SingleSelect  
#  
> 💁 **o1lab**  
####   
![image0](https://cdn.discordapp.com/attachments/1077863293395415071/1077926899306725506/Screenshot_2023-02-22_at_12.16.25.png)  
#  
> 💁 **o1lab**  
#### from left top menu - copy-paste project info  
#  
> 🙋‍♂️ **Hirslay**  
#### Node: **v16.17.1**   
#### Arch: **x64**   
#### Platform: **linux**   
#### Docker: **true**   
#### RootDB: **mysql2**   
#### PackageVersion: **0.105.0**  
#  
> 💁 **o1lab**  
#### upgrade to latest and chk  
#  
> 🙋‍♂️ **Hirslay**  
#### I've updated noco, but error still occurs  
#  
> 🙋‍♂️ **Hirslay**  
#### <@310157751395287040> the only problem is in pop-up window, when I select exact table everything works just fine (but workflow is terrible if we cannot set things from pop-up window 😦  
#  
> 💁 **o1lab**  
#### <@682854952657616902>  
#  
> 🙋‍♂️ **Hirslay**  
#### wanted to share file with logs, but for some reason I cannot share it here  
#  
> 👨‍💻 **pranavxc**  
#### Let me check it  
#  
> 👨‍💻 **pranavxc**  
#### able to reproduce I will work on it, once fixed I will let you know and share a custom docker image to verify.  
#  
> 👨‍💻 **pranavxc**  
#### Fixed here - https://github.com/nocodb/nocodb/pull/5194  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1077863293395415071)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
